web/src/main/angular/src/app/core/components/server-map/class/server-map-diagram-with-cytoscapejs.class.ts

```diff
@@ -346,6 +350,27 @@ export class ServerMapDiagramWithCytoscapejs extends ServerMapDiagram {
         this.baseApplicationKey = baseApplicationKey;
     }
 
+    getResponseInfo(linkData: ILinkInfo|any): any {
+        if (typeof linkData === 'undefined' || !linkData) {
+            return '';
+        }
+        const histogram_avg = linkData.responseStatistics ? linkData.responseStatistics.Avg : 0;
+        if (histogram_avg > 0) {
+            return ' (' + this.formatTime(histogram_avg) + ')';
+        }
+        return '';
+    }
+
+    private formatTime(val: number): string {
+        if (val < 1000) {
+            return val + ' ms';
+        }
+        if (val % 60000 == 0) {
+            return (val / 60000) + ' min';
+        }
+        return (val / 1000.0).toFixed(2).replace('.00', '').replace(/(\.\d)0$/, '$1') + ' sec';
+    }
+
```

Comment:
1. Add line-break after `if` statement and variable declaration.
2. Use template literals when concatenating strings.
3. Use conditional ternary operator to make the code concise.

```suggestion
    getResponseInfo(linkData: ILinkInfo|any): any {
        if (typeof linkData === 'undefined' || !linkData) {
            return '';
        }

        const histogramAvg = linkData.responseStatistics ? linkData.responseStatistics.Avg : 0;

        return histogramAvg > 0 ? ` (${this.formatTime(histogramAvg)})` : '';
    }

    private formatTime(val: number): string {
        return val < 1000 ? `${val} ms`
            : val % 60000 === 0 ? `${(val / 60000)} min`
            : `${(val / 1000.0).toFixed(2).replace('.00', '').replace(/(\.\d)0$/, '$1')} sec`;
    }
```